src/app/translate.ts

```diff
@@ -261,6 +261,8 @@ export class Language {
   organizationName = 'שם הארגון';
   smsMessageContentCaption = 'תוכן הודעת SMS';
   smsReminderMessageContentCaption = 'תוכן הודעת תזכורת SMS';
+  emailDonorContentCaption = 'תוכן אימייל תורם';
```

Comment:
I prefer terms in the term file will refer to family and not donor - donor is specific for mlt